core/tests/protractor_utils/forms.js

```diff
@@ -61,10 +64,16 @@ var GraphEditor = function(graphInputContainer) {
   var createEdge = async function(vertexIndex1, vertexIndex2) {
     var addEdgeButton = graphInputContainer.element(
       by.css('.protractor-test-Add-Edge-button'));
-    await addEdgeButton.click();
+    await action.click('Add Edge Button', addEdgeButton);
+    await waitFor.visibilityOf(
+      vertexElement(
+        vertexIndex1), `Vertex element ${vertexIndex1} taking too long`);
```

Comment:
I think this indentation is easier to read:

```suggestion
    await waitFor.visibilityOf(
      vertexElement(vertexIndex1),
      `Vertex element ${vertexIndex1} taking too long`);
```

Also, `taking too long` is a little vague. Taking too long to do what?